apps/condo/domains/common/components/IFrame/IFrame.tsx

```diff
@@ -84,6 +87,17 @@ export const IFrame: React.FC<IFrameProps> = (props) => {
         setFrameHeight(message.height)
     }, [])
 
+    const handleCommand = useCallback((message) => {
+        const iFrameReceiver = get(iFrameRef, 'current.contentWindow', null)
+        switch (message.command) {
+            case 'getOrganization':
```

Comment:
May be add default case with something like `Not implemented error`?

Reply:
I omitted this intentionally to reduce future refactoring. For now this is only internal code.